dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/metrics/BatchMetricsDispatcher.kt

```diff
@@ -218,6 +238,15 @@ internal class BatchMetricsDispatcher(
         /* The value for the type of the metric.*/
         internal const val BATCH_CLOSED_TYPE_VALUE = "batch closed"
 
+        /* The value of the tracking consent according with this file origin.*/
+        internal const val TRACKING_CONSENT_KEY = "consent"
+
+        /* The file name.*/
+        internal const val FILE_NAME = "name"
+
+        /* The thread name from which the current metric was sent.*/
+        internal const val THREAD_NAME = "thread"
```

Comment:
minor: are these key names taken from a telemetry rfc doc? If we have flexibility it may be better to have more explicit map keys (e.g. it looks like we're sending a semiambiguous `name` in the map, where it might be preferable to send `filename`, and to a lesser extent for thread/threadname). wdyt?

Reply:
wanted to use a shorter key for having lighter logs but yeah...3 chars will not kill